source/memory/memory.go

```diff
@@ -49,5 +68,20 @@ func (s Source) Discover() (source.Features, error) {
 		features["numa"] = true
 	}
 
+	// Check transparent_hugepage
+	choice, err := GetChoice("/host-sys/kernel/mm/transparent_hugepage/enabled")
+	if err != nil {
+		logger.Printf("ERROR: can't read /host-sys/kernel/mm/transparent_hugepage/enabled: %s", err)
+	} else {
+		features["transparent_hugepage.enabled"] = choice
+	}
+
+	choice, err = GetChoice("/host-sys/kernel/mm/transparent_hugepage/defrag")
+	if err != nil {
+		logger.Printf("ERROR: can't read /host-sys/kernel/mm/transparent_hugepage/defrag: %s", err)
+	} else {
+		features["transparent_hugepage.defrag"] = choice
+	}
+
```

Comment:
nit: I think you could reduce the LOC (and possibly easier to add new attributes) by doing a for loop like
```
for _, attrib := range []string{"enabled", "defrag"} {
...
```

What do you think

Reply:
Done.